humanitec-resource-defs/azure-blob/basic/main.tf

```diff
@@ -4,27 +4,30 @@ resource "humanitec_resource_definition" "main" {
   name        = "${var.prefix}azure-blob-storage-basic"
   type        = "azure-blob"
 
+  driver_account = var.driver_account
   driver_inputs = {
-    secrets_string = jsonencode({
-      variables = {
-        client_id     = var.client_id
-        client_secret = var.client_secret
-      }
-    })
-
     values_string = jsonencode({
       source = {
         path = "modules/azure-blob/basic"
         rev  = var.resource_packs_azure_rev
         url  = var.resource_packs_azure_url
       }
 
+      append_logs_to_error = var.append_logs_to_error
+
+      credentials_config = {
+        environment = {
+          ARM_CLIENT_ID     = "appId"
+          ARM_CLIENT_SECRET = "password"
+          ARM_TENANT_ID     = "tenant"
+        }
+      }
```

Comment:
We use environment variables here instead of setting explicit vars in TF as this allows to change the authentication strategy later without adjusting all the modules again.

Reply:
Out of curiosity, this bit should be updated (or we should add a new example) once Azure temporary credentials will be supported?

Reply:
Yes, we generally try to follow best practices here, so I guess we would just update these.